src/Controllers/HomeController.cs

```diff
@@ -4,15 +4,26 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Localization;
 using src.Models;
 
 namespace src.Controllers
 {
     public class HomeController : Controller
     {
+        private readonly IHtmlLocalizer<HomeController> _localizer;
+
+        public HomeController(IHtmlLocalizer<HomeController> localizer)
+        {
+            this._localizer = localizer;
+        }
         public IActionResult Index() => View();
 
-        public IActionResult Privacy() => View();
+        public IActionResult Privacy()
+        {
+            ViewData["Message"] = _localizer["PrivacyPolicy"];

```

Comment:
its ok, but we set a property to access to private attributes, like:

`public IHtmlLocalizer<HomeController> { get => this._localizer; }`